include/dba.php

```diff
@@ -184,7 +186,7 @@ public static function database_name() {
 	private static function logIndex($query) {
 		$a = get_app();
 
-		if (empty($a->config["system"]["db_log_index"])) {
+		if (empty(Config::get('system', 'db_log_index'))) {
```

Comment:
Here is a ```Config``` call. I guess it is not intended?

Reply:
You're right, I'll fix it.